src/app/Models/Traits/HasIdentifiableAttribute.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Backpack\CRUD\app\Models\Traits;
+
+trait HasIdentifiableAttribute
```

Comment:
Indeed, that's a good way to look at it too. 

The ```HasIdentifiableAttribute``` name idea was just to keep things in line [the original traits on the Eloquent Model](https://laravel.com/api/6.x/Illuminate/Database/Eloquent/Model.html): ```HasAttributes```, ```HasEvents```, ```HasGlobalScopes```, ```HasRelationships```, ```HasTimestamps``` and with how other Laravel packages that we use name their traits (ex: the translation package uses ```HasTranslations```).

It would look cleaner to have all traits on the model start with "Has". If we choose to go that way, I would also (someday) rename ```CrudTrait``` into ```HasCrud``` / ```HasCrudPanel``` / ```HasAdminPanel```. So that, in the end, a model would look like this:

```php
class Product extends Model
{
    use HasAdminPanel, HasTranslations;
```

Of course, other packages (like the ones for auditing - ```venturecraft/revisionable``` or ```owen-it/laravel-auditing```) do NOT use this convention. So using one of those would still look out of place 😄 But hey - at least ours would look "Laravelly" 😆

Reply:
I don't know. Let's keep this open and think about it some more.

Reply:
You also have a valid point here. I think with your future plans on renaming we should keep HasIdentifiable as you mentioned first. :)